skins/flux.scss

```
@@ -0,0 +1,10 @@
+.example {

```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.flux {

```